src/LiveComponent/src/Resources/views/form_theme.html.twig

```diff
@@ -0,0 +1,35 @@
+{%- block live_collection_widget -%}
+    {% set attr = attr|merge({'class': attr.class|default('space-y-4')}) %}
+    <div {{ block('widget_container_attributes') }}>
+        {%- if form is rootform -%}
+            {{ form_errors(form) }}
+        {%- endif -%}
+        {{- block('form_rows') -}}
+        {{- form_rest(form) -}}
+
+        {% if button_add_prototype is defined and not button_add_prototype.rendered %}
+            {{ form_row(button_add_prototype, { attr: button_add_prototype.vars.attr|merge({
+                'data-action': 'live#action',
+                'data-action-name': 'addCollectionItem(name=' ~ form.vars.full_name ~ ')'
+            }) }) }}
+        {% endif %}
```

Comment:
It's funny, #90 adds this at the top of the list, and you have it at the bottom. Nothing will work for everyone, but the bottom I think DOES seem better.

Reply:
Let's start with one... but it occurs to me that it may be be useful to configure this easily (e.g. `top`, `bottom` or `both`)